joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -350,6 +350,12 @@ controller_interface::return_type JointTrajectoryController::update(
       }
     }
   }
+  else
+  {
+    // TODO(peterdavidfagan) temporary solution to keep the initial position
```

Comment:
There is nothing more permanent then a temporary solution ;P
I put the comment there as I was unsure what kind of implications this has for other use-cases.

Reply:
I am happy to continue working on this pr based on feedback from ros-controls developers and my evolving understanding of the scripts.